### PR TITLE
Release/0.1

### PR DESCRIPTION
**Pull request by cowman19**
Initial setup Fixed Wing symbol library, and creation of LTC6804-2 symbol.

### CODE REVIEW

**Review by colajacquin-e — CHANGES_REQUESTED** [review]
aye that thing is weird man

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## User story or task linked to this change
+<!--- Add the DevOps user story or task number and title here -->
```
Comment (colajacquin-e) [inline .github/pull_request_template.md:4]: this should be capitalized

Comment (cowman19) [reply .github/pull_request_template.md:4]: 'mongus

Comment (cowman19) [reply .github/pull_request_template.md:4]: Issue fixed in commit 78b0d33

**Review by colajacquin-e — APPROVED** [review]

**Review by dhruvdingari8 — COMMENTED** [review]
goated :thumbs_up:

**Pull request merged**